### ISSUE TITLE
Definition Error Missing Audio File

### ISSUE THREAD

**Issue by kuroodo** [KeenSoftwareHouse/SpaceEngineers#7]
I keep getting a definition error after running or debugging the project.

Unable to find audio file: 'RealWepPlayRifleImpMetal', 'REAL\IMP\RealWepPlayRifleImpMetal3d01.wav'.

I looked for the wav file myself in the game directory and was unable to find it. I did find the .xwm file though. In the audio.sbc file, I tried deleting the entry for the xwm file but I still received the error.

**Comment by OndrejPetrzilka** [comment]
TLDR;
Edit **global.props** (or **user.props**), set DataFromSources to false.

**Assert: unable to find audio/model/texture file: 'xxxxxx'**.
This happens because repository is slightly ahead of content in Steam folder. Definitions (Content/Data) are taken from repository and may contain new definitions referencing assets which are not yet in Steam content folder. We decided to use definitions from repository by default, so you can easily modify it. You can edit **global.props** to use definitions from Steam (that should fix the issue). When running on **Release** asserts won't be shown; missing assets won't crash the game.  More info [here](https://github.com/KeenSoftwareHouse/SpaceEngineers/wiki/Initial-setup#setting-path-to-the-games-content).
